### ISSUE TITLE
SereneSeasons "Season Clock" can't make up its mind.

### ISSUE THREAD

**Issue by ghost** [SpongePowered/SpongeForge#3004]
**I am currently running**

- SpongeForge version: 1.12.2-2838-7.1.7-RC3928
- Forge version: (1.12.2) 14.23.5.2838
- Java version: 1.8.0_211/64 bit
- Operating System: Windows 10

- Plugins/Mods: Homestead Beta Modpack ( https://www.technicpack.net/modpack/homestead-smp.1479367 )
I'm not going to bother putting all the mods and stuff here, as I am already aware this is a bug with SpongeForge.

**Issue Description**
Holding a Season Clock from the Mod "SereneSeasons" on a SpongeForge server is the most confusing task.

You want to know the season? Good luck, because it will spin on every season and will never stop on one. Think of it like a clock that makes a day last 2 minutes, oh and backwards.

Basically, the Season Clock is supposed to tell you which season it is (in the form of a clock)
But instead of working like a Minecraft clock (But for seasons!) it spins randomly and doesn't stop until the end of time.

I'm pretty sure this is a SpongeForge issue as SereneSeasons works completely fine in Singleplayer, but not on my server.

Thanks in advance!

https://www.youtube.com/watch?v=S7AfOWDOKhc

**Comment by JBYoshi** [comment]
I can't reproduce this on the latest SpongeForge, although I'm only using that and Serene Seasons.

One quick question: Are you using any plugins, or just mods and SpongeForge?

**Comment by ghost** [comment]
Hey there, @JBYoshi 

I am currently on a SpongeForge Server with several plugins. However, a different setup.
The server has two worlds- the spawn world (sponge:the_void) and the main world (biomesop)

After some more investigating, it appears that the clock freaks out only in the main world, not the spawn world- this was intentional by the mod author.

There is a configuration option in SereneSeasons to enable these Season Changes in "whitelisted" dimensions.

I apologise for not closing this issue sooner- I didn't get around to making an explanation as to why.
I presumed that the clock spinning out of control (intentional by mod author) was a bug to do with SF.

Cheers and all the best.
